<launch>

    <node name="sparse_to_dense" pkg="sparse_to_dense" type="main.py" output="screen" required="true" 
      args="--ros">

        <param name="model_path" value="$(find sparse_to_dense)/results/nyudepthv2.sparsifier=uar.samples=100.modality=rgbd.arch=resnet18.decoder=upproj.criterion=l1.lr=0.01.bs=8.pretrained=True.varFocus=False.varScale=False/model_best.pth.tar" />
        <param name="rate" value="10" />
        <remap from="rgb_in" to="/camera/rgb/image_color" />
        <remap from="depth_target" to="/camera/depth/image" />
    </node>

</launch>